### PR TITLE
add contrast sensitivity functions basic frame

### PR DESCRIPTION
**Pull request by Page-David**
Hi, I'd like to better the issue #71. I finally find some English material (Lisa J. Croner, Thomas Wachtler (auth.), Christian J. van den Branden Lambrecht (eds.)-Vision Models and Applications to Image and Video Processing-Springer US (2001)) describes this phenomenon. From that book, I found it should be explained with experiment as well, but sadly, there is no detail process. Also, here is a picture in that book, although I think it may be illegal used in this free project, but I'd like post here and if we find an alternative of it, we can attach it to the article.
2018-08-12-145553_1920x1080_scrot

### CODE REVIEW

**Comment by leandromoreira** [comment]
Maybe we can add this as a more in depth explanation but the **simplistic** still might help

**Comment by Page-David** [comment]
I copied the explanation back into the file. And.. it might bother you to reformat a bit.

File: README.md [hunk 1]
```diff
@@ -171,14 +171,17 @@ Our eyes are [more sensitive to brightness than colors](http://vanseodesign.com/
...
 >
 > Since we have many more rod cells (brightness) than cone cells (color), one can infer that we are more capable of distinguishing dark and light than colors.
 >
 > ![eyes composition](/i/eyes.jpg "eyes composition")
+>
+> **contrast sensitivity functions**
+>
+> Researchers of experimental psychology and many other fileds have set up many theories on human vision. And one of them is called contrast sensitivity functions. They are related to sptial and temporal of the light we see and their value presents at given init light, how much relative change of it required before an observer reports he realizes a change takes place. Notice the plural of the word "function", it is for the reason that we can measure contrast sensitivity functions with not only black-white but also colors. The result of experiment shows that in most cases our eyes are more sensitive to achromatic than chromatic.
```
Comment (leandromoreira) [inline README.md]: Please fix the following misspelled words: **fileds**, **sptial** and **experiment** (this I think you just missed the article THE)

Comment (matteocontrini) [reply README.md]: I'd also change the first letter in **contrast** so that it's capitalized.

**Comment by Page-David** [comment]
Sorry for the type errors, I have fixed them and removed some words that seems to be unnecessary.

**Pull request merged**